s2and/extentions/featurization/utils.py

```diff
@@ -75,22 +73,22 @@ def jaccard(
 def featurizing_function(
     sig1: Dict[str, dict],
     sig2: Dict[str, dict]
-) -> float:
+) -> List[float]:
     """
     Calculates the feature vector of two given signature dicts
     """
     features = []
     features.append(cosine_sim(sig1, sig2))
     features.append(name_distance(sig1, sig2))
-    features.append(jaccard(sig1, sig2, 'affiliationIds'))
-    features.append(jaccard(sig1, sig2, 'fosIdsLevel0'))
-    features.append(jaccard(sig1, sig2, 'fosIdsLevel1'))
-    features.append(jaccard(sig1, sig2, 'authorNormNames'))
+    features.append(jaccard(sig1, sig2, 'affiliationsIds'))
+    features.append(jaccard(sig1, sig2, 'OAfos_0'))
+    features.append(jaccard(sig1, sig2, 'OAfos_1'))
+    features.append(jaccard(sig1, sig2, 'coAuthorShortNormNames'))
 
     # Ids for later use from the ensemble
-    features.append(extract_feature(sig1, sig2, 'magAuthorId'))
-    features.append(extract_feature(sig1, sig2, 's2AuthorId'))
-    features.append(extract_feature(sig1, sig2, 'orcId'))
+    features.append(extract_feature(sig1, sig2, 'OAauthorId'))
+    features.append(extract_feature(sig1, sig2, 'S2authorId'))
+    features.append(extract_feature(sig1, sig2, 'orcid'))
```

Comment:
Unfortunately this has to be flexible because we experiment with the input features of the pairwise classifier. Given that, I cannot fix the logic with some yaml file.

Reply:
That's alright, no problem.